### ISSUE TITLE
"extract" command has no destination switch?

### ISSUE THREAD

**Issue by jumper444** [jborg/attic#195]
Extract command seems to have no destination option.  It drops files into the current working directory (recreating their directory structure when saved). 

I've never seen this before.  I've never seen a backup, zip, archive, or other 'save it and restore it' program that didn't allow the extract or restoration of files to have a specified destination.
Am I missing something?

**Comment by devZer0** [comment]
looks weird, indeed - but you can always cwd into the appropriate directory before extracting. at least a workaround, but makes scripts more complex, though - as you need another command line

**Comment by jumper444** [comment]
I'm not as versed enough in linux to fully defend my comment here, but there are generally multiple ways of attaching/mounting or sending/receiving to remote locations from the command line.  However I'm not aware that all (most? any?) of these methods allow a cwd to that location first.  Thus, perhaps, the inability of having an extract location might force two full data copy operations instead of one: extract to a local (cwd possible) location and then a copy/move to a remote destination?  Yes?

I guess whatever issue I might be raising hasn't been hit by the majority of Attic users so there seems to be no critical problem. I just was surprised you couldn't say 'put my files [here]' when taking them out.

**Comment by nonsub** [comment]
Agreed jumper... I have to do some test recoveries, if I figure out some methods like cwd etc I'll post here.
